commands/utils/params.go

```diff
@@ -85,9 +89,27 @@ func GetParamsAndClient() (configAggregator FrogbotConfigAggregator, server *cor
 		return nil, nil, nil, err
 	}
 
-	configData, err := ReadConfig(frogbotConfigPath)
+	var targetConfigContent []byte
+	var configData *FrogbotConfigAggregator
+	var missingConfigErr bool
+	if gitParams.GitProvider.String() == vcsutils.GitHub.String() || gitParams.GitProvider.String() == vcsutils.GitLab.String() {
+		targetConfigContent, err = DownloadConfigFromTarget(client)
+		_, missingConfigErr = err.(*ErrMissingConfig)
+		if err != nil && !missingConfigErr {
+			return nil, nil, nil, err
+		}
+		if targetConfigContent != nil {
+			if err = yaml.Unmarshal(targetConfigContent, &configData); err != nil {
+				return nil, nil, nil, err
+			}
+		}
+	}
+	if targetConfigContent == nil {
+		configData, err = ReadConfig(frogbotConfigPath)
+		_, missingConfigErr = err.(*ErrMissingConfig)
+	}
```

Comment:
I suggest moving the above line into a "readConfig" function.

Reply:
done